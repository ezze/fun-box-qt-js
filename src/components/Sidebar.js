import React from 'react';

const Sidebar = () => {
    return (
        <div className="sidebar">
            <header className="sidebar-header">Маршрут</header>
        </div>
    );
};

export default Sidebar;
