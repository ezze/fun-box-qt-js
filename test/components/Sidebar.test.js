import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Sidebar from '../../src/components/Sidebar';

describe('sidebar component', () => {
    it('rendering', () => {
        const renderer = new ShallowRenderer();
        renderer.render(<Sidebar />);
        const tree = renderer.getRenderOutput();
        expect(tree).toMatchSnapshot();
    });
});
