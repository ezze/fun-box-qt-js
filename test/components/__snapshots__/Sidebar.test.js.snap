// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sidebar component rendering 1`] = `
<div
  className="sidebar"
>
  <div
    className="sidebar-header"
  >
    Маршрут
  </div>
  <div
    className="sidebar-content"
  >
    <Connect(RouteForm) />
    <Connect(RouteList) />
  </div>
</div>
`;
